osCam/userconfig/forms.py

```diff
@@ -27,6 +27,12 @@ class Meta:
         model = Camera 
         fields = ('model_info',)
 
+# alternate carera form whoops
+# class CameraForm(forms.ModelForm):
+#     class Meta():
+#         model = Camera
+#         fields = ('recording', 'fps', 'invert', 'mirror', 'codec', 'camera_index', 'device_name', 'ip_address', 'port', 'username', 'password')
```

Comment:
I think we should also add 'location' or 'camera_name' so that when it is displayed, we can also display the 'location' of the camera as entered by the user such as "front door", "back yard" etc.

Reply:
i believe this is somewhere and this was an in-progress form that did not work. 

honestly shoulda been left outta the commit.